src/FormContainer.tsx

```diff
@@ -92,7 +92,7 @@ const makeWrapper = <T extends {}>(config: IFormConfig) => (WrappedComponent: an
         });
 
         render() {
-            const nextProps = Object.assign({}, this.props, {
+            const nextProps: any = Object.assign({}, this.props, {
```

Comment:
This `any` doesn't feel right. I think we need to address the overuse of `any` throughout this repo.

Reply:
agreed, but I couldn't find the right type for it that does not throw an error. Thoughts?

Reply:
Leave it for now, but let's address it as part of a wider issue. 🙂